index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
```

Comment:
Não era um requisito, mas ficaria muito bom se você colocasse um título bacana na página, exemplo:
```suggestion
    <title>Lessons Learned por Flavio Fernandes</title>
```